src/useCases/getAccountBeacons.ts

```diff
@@ -0,0 +1,16 @@
+import { IBeacon } from "../entities/beacon";
+import { IAppContainer } from "../lib/appContainer";
+
+export type GetAccountBeaconsFn = (accountId: string) => Promise<IBeacon[]>;
+
+export const getAccountBeacons =
```

Comment:
Suggestions for a more specific name here: `getAllBeaconsByAccountHolderId` or `getBeaconsByAccountHolder` or `getBeaconsByAccountHolderId`?